src/App.js

```diff
@@ -1,9 +1,20 @@
+import { useState, useEffect } from 'react'
+import data from './data/spotifyTrackData.json'
+import normalizeArtists from './services/normalizeArtists'
+import ArtistList from './components/ArtistList'
+
+export default App
+
 function App() {
+  const [artistData, setArtistData] = useState([])
+
+  useEffect(() => {
+    setArtistData(normalizeArtists(data.tracks.items))
+  }, [])
```

Comment:
Consider puting line 8-13 into a costum hook and return -> const {artistData} = useArtists()

Reply:
The state is not very complex at this point. For now, I find it clearer to have it directly in the app. I'll keep it in mind for later. Thank you!